docs/en/siem/rules-api-create.asciidoc

```diff
@@ -0,0 +1,117 @@
+:api-call: create-rule
+[[rules-api-create]]
+=== Create rule
+
+Creates a new signal detection rule.
+
+==== Request URL
+
+`POST /api/detection_engine/rules`
+
+==== Request body
+
+A JSON object with these fields:
+
+include::shared-content/rule-schema.asciidoc[tag=rules-api-json-schema]
+
+===== Example request
+
+Searches for processes started by MS Office: 
+
+[source,js]
+--------------------------------------------------
+POST api/detection_engine/rules
+{
+  "rule_id": "process_started_by_ms_office_program_possible_payload",
+  "risk_score": 50,
+  "description": "Process started by MS Office program - possible payload",
+  "interval": "1h", <1>
+  "name": "MS Office child process",
+  "severity": "low",
+  "tags": [
+   "child process",
+   "ms office"
+   ],
+  "type": "query",
+  "from": "now-4200s", <2>
+  "query": "process.parent.name:EXCEL.EXE or process.parent.name:MSPUB.EXE or process.parent.name:OUTLOOK.EXE or process.parent.name:POWERPNT.EXE or process.parent.name:VISIO.EXE or process.parent.name:WINWORD.EXE",
+  "language": "kuery",
+  "filters": [
+     {
+      "query": {
+         "match": {
+            "event.action": {
+               "query": "Process Create (rule: ProcessCreate)",
+               "type": "phrase"
+            }
+         }
+      }
+     }
+  ],
+  "enabled": false
+}
+--------------------------------------------------
+// KIBANA
+
+<1> The rule runs every hour.
+<2> When the rule runs it analyzes data from 70 minutes before its start time.
```

Comment:
Should we leave a link to the date math docs here? https://www.elastic.co/guide/en/elasticsearch/reference/current/common-options.html#date-math

Reply:
@dhurley14 
This page includes the JSON rule schema table, which contains a link to the date maths docs: http://stack-docs_699.docs-preview.app.elstc.co/guide/en/siem/guide/master/rules-api-create.html

Here's a [link](http://stack-docs_699.docs-preview.app.elstc.co/guide/en/siem/guide/master/rule-api-overview.html) to a preview of the docs in the first comment, which displays how they'll appear when they are published.